tests/components/aemet/test_sensor.py

```diff
@@ -37,9 +37,7 @@ async def test_aemet_forecast_create_sensors(hass):
     assert state.state == "-4"
 
     state = hass.states.get("sensor.aemet_daily_forecast_time")
-    assert (
-        state.state == dt_util.parse_datetime("2021-01-10 00:00:00+00:00").isoformat()
-    )
+    assert state.state == dt_util.parse_datetime("2021-01-10 00:00:00+00:00")
```

Comment:
The state will still be an isoformatted string. The base sensor class takes care of it now.